components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md

```diff
@@ -92,9 +93,9 @@ You can also generate a configuration file using the `init-config` subcommand.
     sudo chef-automate verify -c config.toml
     ```
 
-    To know more about config verify you can check [Config Verify Doc page](/automate/ha_verification_check/).
+    To know more about config verify, check [Config Verify Documentation](/automate/ha_verification_check/).
 
-    Once the verification is successfully completed, then proceed with deployment, In case of failure please fix the issue and re-run the verify command.
+    Once the verification is completed successfully, proceed with the deployment. In case of failure, fix the issue and and verify it by re-running the verify command.
```

Comment:
Once the verification completed successfully

Reply:
and verify it by re-running

Reply:
Done